modules/waf/main.tf

```diff
@@ -16,7 +16,7 @@ locals {
       }
     }]
     : [],
-    var.rules
+    var.enable_default_rule ? local.default_rules : var.rules
```

Comment:
maybe better to join externally passed rules with default rules instead of having switch from default to passed rules?